litmus-portal/frontend/src/pages/Community/styles.ts

```diff
@@ -53,6 +53,16 @@ const useStyles = makeStyles((theme) => ({
       flexDirection: 'column',
     },
   },
+  paperTimeSeries: {
+    width: theme.spacing(80),
+    height: theme.spacing(57.45),
+    color: '#FFFFFF',
```

Comment:
theme.palatte.common.white

Reply:
All changes suggested have been made please verify the same.